### DISCUSSION TITLE
Trio support

### DISCUSSION THREAD

**Discussion by lordmauve** [hynek/stamina#42] [General]
Does stamina support Trio?

Tenacity is documented as doing so if you pass the correct sleep function, i.e. `@retry(sleep=trio.sleep)`. But stamina doesn't offer the API so I'm guessing no?

I think anyio would solve this? Or if a whole extra dependency is too heavyweight, vendor the slice of anyio that resolves the correct sleep function to use?

**Comment by hynek** [comment]
c.f. #43

**Comment by hynek** [comment]
23.3.0 with Trio support is on PyPI: #44

**Reply by lordmauve** [reply]
Wow! Thanks! Sorry I haven't been paying attention, I'm on a Weihnachtsmarkts trip in Stuttgart

**Reply by hynek** [reply]
Dive into the Glühwein!

**Reply by lordmauve** [reply]
✅ Glühwein
✅ Germknödeln
✅ Flammlachs
✅ Eierpunch
✅ Bratwurst
